pkg/engine/masterarmresources.go

```diff
@@ -42,7 +42,7 @@ func createKubernetesMasterResourcesVMAS(cs *api.ContainerService) []interface{}
 	kubernetesConfig := cs.Properties.OrchestratorProfile.KubernetesConfig
 
 	if !cs.Properties.OrchestratorProfile.IsPrivateCluster() {
-		publicIPAddress := CreatePublicIPAddress()
+		publicIPAddress := CreatePublicIPAddress(true)
```

Comment:
nit: assign a variable like `isMaster := true` and pass `isMaster` to `CreatePublicIPAddress` so it's clearer to future maintainers what that bool arg is for.